README.md

```diff
@@ -170,10 +168,7 @@ SMU v46.54.0
 2
 
 # cat /sys/kernel/ryzen_smu_drv/codename
-4
-
-# cat /sys/kernel/ryzen_smu_drv/drv_version
-0.1.7
+StrixPoint
```

Comment:
I don't see where the codename endpoint (string) is implemented. But if it is, its an API break which requires more than a minor patchlevel bump.

Reply:
leftover from a reverted change, sorry
fixed in latest commit